ArtOfIllusion/src/artofillusion/ui/TreeList.java

```diff
@@ -479,7 +479,7 @@ private void mousePressed(MousePressedEvent ev)
       showPopupIfNeeded(ev);
       return;
     }
-    if (ev.isShiftDown() || ev.isControlDown() || ev.isMetaDown())
+    if (ev.isShiftDown() || mouseButtonTwo(ev) || mouseButtonThree(ev))
```

Comment:
Two mousebuttons simultaneously.

```suggestion
     if (ev.isShiftDown() ||  ev.isControlDown() || mouseButtonThree(ev))
```
The control key is used to pick single (not consequently listed) objects for an action like texturing or making a boolean operation.

Reply:
Yeah, this one should not have been changed at all. Fixed.